js/signin.js

```diff
@@ -77,28 +84,35 @@ function formSubmit(event) {
 
   const email = emailInput.value.trim();
   const password = passwordInput.value.trim();
+  const testEmail = "test@codeit.com";
+  const testPassword = "codeit101";
```

Comment:
요렇게 상수임을 조금 더 명확하게 표현해주시면 좋을 것 같아요!

```suggestion
  const TEST_EMAIL = "test@codeit.com";
  const TEST_PASSWORD = "codeit101";
```